python/setup.py.in

```diff
@@ -0,0 +1,9 @@
+#!@PYTHON@
+
+from distutils.core import setup
```

Comment:
FYI, distutils will be deprecated soon. https://peps.python.org/pep-0632/

I'm not familiar with Python packaging and so I don't know off the top of my head what, if any, alternative should be used. I remembered seeing this deprecation notice a while back though.